Extending/Macro-Parameter-Editors/index.md

```diff
@@ -116,57 +115,70 @@ angular.module("umbraco").controller("tooorangey.ImagePositionController", funct
 
 The final custom parameter should look like this:
 
-![Image Position Radio Button Options](images/image-position.png)
+![Image Position Radio Button Options](images/image-position-v8.png)
 
-## Umbraco 6 - Creating your own macro parameter type ##
+#### Using defaultConfig
 
-If you want to create a new macro parameter editor you will need some c# programming and database knowledge.
+In this example it doesn't really add anything to move the radio button options into configuration, however to illustrate the concept of providing defaultConfig, let's do just that:
```

Comment:
```suggestion
In this example it doesn't really add anything to move the radio button options into configuration, however to illustrate the concept of providing defaultConfig, let's do that:
```

Reply:
Ha ha ... that's not a 'just' in the sense of 'you just do.... blah de blah' without going on to explain what the thing is... but I'm happy for zero tolerance on the word!

Reply:
Yeah, I see your point 😅 
Let's go with the zero tolerance. I mean, it can easily be understood without the "just" .. of course, I do get that it doesn't have quite the same effect without it 🤔

Reply:
(well it will just keep showing up on our reports if we just leave it in....)